### DISCUSSION TITLE
How to do repeated questions?

### DISCUSSION THREAD

**Discussion by v3ss0n** [copier-org/copier#1236] [Q&A]
For example like this :

 - Add models?
 - Yes
 - Give a model Name
 - `employee`
  - Add models?
 - Yes
  - Give a model Name
 - `team`
 
 ...
So that , user can create models repeateldy.

**Comment by sisp** [comment]
This is related to https://github.com/copier-org/copier/issues/750. Unfortunately, it isn't possible yet.

**Reply by v3ss0n** [reply]
how about adding an list type? where it will be list of entries?

**Reply by sisp** [reply]
Would a `type: yaml` or `type: json` question work, too? The answer would be an array of strings in your case which you can enforce through the `validator` field with custom logic or using the [`jinja2-jsonschema`](https://github.com/copier-org/jinja2-jsonschema) extension.

**Reply by v3ss0n** [reply]
I gonna try that  , that could work.  But user would need to install extra extension for validation? 
also , any way to generate list of files according to that rules? just looping in jinja file name would work?

**Reply by sisp** [reply]
> But user would need to install extra extension for validation?

Only if you want to use the `jinja2-jsonschema` extension, but if you stick with a manual Jinja2 rule (which shouldn't be very difficult to write in this case), you can get away without an extension.

> also , any way to generate list of files according to that rules? just looping in jinja file name would work?

I'm afraid I don't understand this question. :slightly_frowning_face:

**Reply by v3ss0n** [reply]
I was looking for an example.
Lets say i have this in json  `{ 'domains' : [ "books","authors" , "users" ]}` , i want to create each of them  in directory :

```
domains/
-----books.py
-----authors.py
-----users.py
```

**Reply by sisp** [reply]
Ah, that's actually more related to https://github.com/copier-org/copier/discussions/908. I proposed a feature design in https://github.com/copier-org/copier/discussions/908#discussioncomment-5466993 but haven't had the time to work on it. But I think it would do what you need.

**Reply by v3ss0n** [reply]
I see, so currently it's  not in
